.gitlab-ci.yml

```diff
@@ -37,80 +37,80 @@ before_script:
   <<: *build_definition
   variables:
     OS: 'alpine'
-    OS_VER: '3.5'
+    DIST: '3.5'
```

Comment:
This variable was introduced right in this PR. There is no reason to give it OS_VER name and then rename to DIST. Let's introduce it as DIST from scratch. (I understand, rebasing is not fun, but see, the result will be more nice.)

The same for BASE_IMAGE.